test/template-create/out/my-template/v0/extractor/ex-generic-v2/om-config/config.jsonnet

```diff
@@ -4,7 +4,7 @@
       tables: [
         {
           source: "table",
-          destination: "in.c-my-super-bucket-om-config.table",
+          destination: "in.c-my-super-bucket-" + ConfigId("om-config") + ".table",
```

Comment:
Zmeny vo funkcionalnom teste:
- tu bolo v projekte nejake numericke ID, takze je to nahradene za `ConfigId(...)`.